scripts/utils.js

```diff
@@ -288,21 +320,50 @@ function buildTagsBlock() {
       recBlock.before(tagsBlock);
     }
   } else {
-    main.lastElementChild.append(tagsBlock);
+    mainEl.lastElementChild.append(tagsBlock);
   }
 }
 
+async function buildArticleMeta(mainEl) {
+  const miloLibs = getLibs();
+  const { getMetadata, getConfig } = await import(`${miloLibs}/utils/utils.js`);
+
+  // author + publication date
+  const author = getMetadata('author') || 'Adobe Communications Team';
```

Comment:
Probably not a big deal, but any localization needed for `Adobe Communications Team`?

Reply:
Thanks for the flag. I looked into the Milo code & Sharepoint, and there's `placeholders.xlsx` in each region folder.
Could I add a new row with key as `adobe-communication-team` and value as `Adobe Communications Team`  in all region's folder's `placeholder.xlsx` for now for the localized text implementation? 

<img width="1032" alt="Screen Shot 2024-09-26 at 4 39 29 PM" src="https://github.com/user-attachments/assets/479bf767-dd08-4e09-bc74-3b0242fd4448">

And afterwards, how can we request the regions to provide the localized text in the `placeholder.xlsx`? Thank you